### ISSUE TITLE
Battery state stream on DeviceController

### ISSUE THREAD

**Issue by LarsRefsgaard** [cph-cachet/carp.sensing-flutter#330]

**Comment by bardram** [comment]
This has now been implemented. There are two options:

1. You can listen to all battery events on the device controller like this:

```
    DeviceController().batteryEvents.listen((event) => print(event));
```

2. You can listen to battery event on each device manager of type `HardwareDeviceManager` like this:

``` 
    manager.batteryEvents;
````

where `manager` is a `HardwareDeviceManager` like the `SmartphoneDeviceManager` or the `ESenseDeviceManager`

**Comment by bardram** [comment]
This, however, is not published yet, but is available in the `develop-1-1-0` branch.

**Comment by bardram** [comment]
Published as part of `1.2.0`.
